src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -160,8 +171,14 @@ def collect_data(geop_results, geojson, watershed_id=None, weather=None):
     z['SedAFactor'] = sed_a_factor(geop_result['landuse_pcts'],
                                    z['CN'], z['AEU'], z['AvKF'], z['AvSlope'])
 
-    z['LS'] = ls_factors(geop_result['lu_stream_pct'], z['StreamLength'],
-                         z['Area'], z['AvSlope'], ag_lscp)
+    # Use zeroed out stream variables if there are no streams in the AoI
+    if 'lu_stream_pct' in geop_result:
+        z['LS'] = ls_factors(geop_result['lu_stream_pct'], z['StreamLength'],
+                             z['Area'], z['AvSlope'], ag_lscp)
+    else:
+        zeroed_lu_stream_pct = [0 for i in range(0, 16)]
```

Comment:
Prefer:

```python
[0.0] * 16
```

for declaring a 16-index zeroed array, as done elsewhere:

https://github.com/WikiWatershed/model-my-watershed/blob/3902fb97e9cc114c0c035a4ec2b5ec2307f86a8e/src/mmw/apps/modeling/mapshed/calcs.py#L273

Reply:
Thanks